spec/bolt/outputter/human_spec.rb

```diff
@@ -249,6 +249,12 @@
     expect(lines).to match(/^    "key": "val"$/)
   end
 
+  it 'prints lookup results' do
+    result = Bolt::Result.for_lookup(target, 'key', 'value')
+    outputter.print_result(result)
+    expect(output.string).to match(/value/)
```

Comment:
```suggestion
    expect(output.string).to eq("Finished on target1:\n  value\n")
```
Or can do `#{target.name}` if you're worried about the name changing, but I wouldn't be.